Sources/Main.cpp

```diff
@@ -681,15 +757,31 @@ void __fastcall TX584Form::CodeListViewMouseDown(TObject *Sender,
     } else {
         Rect.Left += CodeListView->Columns->Items[0]->Width + CodeListView->Columns->Items[1]->Width +
             CodeListView->Columns->Items[2]->Width;
-        if (WasSelected && X > Rect.Left) {
-            //щелкнули в области колонки комментариев - запускаем таймер редактирования
+        Rect.Right = Rect.Left + CodeListView->Columns->Items[3]->Width;
+        if (WasSelected && X >= Rect.Left && X <= Rect.Right) {
+            //щелкнули в области колонки управляющих команд - запускаем таймер редактирования
             EditRow = Row;
+            EditColumn = 2;
             EditPoint.x = CodeListView->Left + Rect.Left + 8;
             EditPoint.y = CodeListView->Top + Rect.Top + 2;
             //перезапускаем таймер
             ClickTimer->Enabled = false;
             ClickTimer->Enabled = true;
         }
+        else {
+            Rect.Left = Rect.Right;
+            Rect.Right += Rect.Left + CodeListView->Columns->Items[4]->Width;
```

Comment:
Shouldn't we use a simple assignment operator `=` instead of `+=` ?

Reply:
Yes. This is some kind of "gotcha" - typo which can lead to a bug